dotnet/src/Microsoft.AutoGen/Agents/IAgentWorkerRuntime.cs

```diff
@@ -6,6 +6,8 @@ namespace Microsoft.AutoGen.Agents;
 public interface IAgentWorkerRuntime
 {
     ValueTask PublishEvent(CloudEvent evt);
```

Comment:
In event-driven pattern, command goes together with event/data so it might be possible that eventually we only need `PublishEvent` and `Store/Read` might be something that can build upon `PublishEvent`

Reply:
you are on to something in that the memory service will build on publish event. this persistence is meant to just provide agent continuity across resets/migrations/etc and to be transparent to the user/handled by Orleans (which under the hood builds this on async events).